Resources/Prototypes/SS220/Catalog/uplink_catalog.yml

```
@@ -169,6 +169,19 @@
   categories:
   - UplinkWearables
 
+- type: listing
+  id: UplinkBorgModuleFlamingAxe
+  name: uplink-borg-module-axe-name
+  description: uplink-borg-module-axe-desc
+  productEntity: BorgModuleFlamingAxe
+  discountCategory: usualDiscounts
+  discountDownTo:
+    Telecrystal: 1
+  cost:
+    Telecrystal: 3
```

```suggestion
    Telecrystal: 2
  cost:
    Telecrystal: 3
```

Слишком сильная штука, можно реально личико человечку поломать, дискаунт до 1 неоправдан.